packages/terra-clinical-site/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ ChangeLog
 
 Unreleased
 ----------
+### Added
+* Added new props to Props Table of Detail View and new example for DetailView with Smaller Title and secondary Titles.
```

Comment:
Looks like a new props table wasn't added. Can we update this to indicate we added a new DetailView example. We can also drop the Changes section since we only added one example

Reply:
Correct, I haven't added a new props table, but I have added secondaryTitles, accessory and isSmallerTitles props to the existing props table.

I will go ahead and drop the changes section.